spring-cloud-dataflow-docs/src/main/asciidoc/howto.adoc

```diff
@@ -16,3 +16,74 @@ We're also more than happy to extend this section; If you want to add a '`how-to
 can send us a {github-code}[pull request].
 --
 
+== Configure Remote Maven Repository
+
+If you'd like to connect to your organization's internal maven repository, you can override the default 
+repository location by passing the credentials via the following environment properties:
+
+```
+export MAVEN_LOCAL_REPOSITORY=mylocalMavenRepo
+export MAVEN_REMOTE_REPOSITORIES_REPO1_URL=https://repo1
```

Comment:
This will not work as mentioned in the previous doc. We should not be using this way of specifying the environment properties.